### ISSUE TITLE
What's this?

### ISSUE THREAD

**Issue by pipboy96** [wellington1993/MyFirefox#1]
Is this just your firefox directory?

**Comment by wellington1993** [comment]
@pipboy96 
Yes... Firefox updates by yourself... and changes a lot of files.

That project just have a bash script.
The  script start the software and do commit of changes...
Just a way to simple control the software changes.

I know is not the official repository but they not using GIT... 

I'm using a Nightly version, to test new features...
With GIT I can reverse to working 'moment' when necessary, waiting for fixes in the next day. Just this.

Did you understand?

Thanks!

**Comment by pipboy96** [comment]
@wellington1993 Yes, I did. Thanks for explaining.

I would personally be worried about privacy posting my browser directory on GH. I would keep it stored locally instead (git repo can be fully local).

**Comment by wellington1993** [comment]
@pipboy96 There isn't problem in make the repository public.
The Firefox profile is located in the home directory(/home/youruser, ~).

It's a good exercise for me. Because if Mozilla change anything more then binaries and non personal files I can tell them to fix.

Thanks!
